hnn_core/gui/_viz_manager.py

```diff
@@ -217,9 +244,31 @@ def _dynamic_rerender(fig):
 
 
 def _plot_on_axes(b, widgets_simulation, widgets_plot_type,
+                  target_simulations,
                   spectrogram_colormap_selection, dipole_smooth,
                   max_spectral_frequency, dipole_scaling, widgets, data,
                   fig_idx, fig, ax, existing_plots):
+    """Plotting different types of data on the given axes.
+    Now this function is also responsible for comparing multiple simulations,
```

Comment:
I like that you started documenting this but could we document this in standard numpy format?

Reply:
updated in the latest commit